mlflow/pyfunc/model.py

```diff
@@ -228,17 +229,18 @@ def _save_model_with_class_artifacts_params(
     if artifacts:
         saved_artifacts_config = {}
         with TempDir() as tmp_artifacts_dir:
-            tmp_artifacts_config = {}
             saved_artifacts_dir_subpath = "artifacts"
```

Comment:
nit: we can move the `saved_artifacts_dir_subpath` definition to the `else` branch below.

Reply:
I don't think we should move it below, there's a for loop.

Reply:
@chenmoneygithub `saved_artifacts_dir_subpath` is used on line 247.

Reply:
Thanks Haru